README.md

```diff
@@ -586,7 +588,7 @@ Support import camera links from [Home Assistant](https://www.home-assistant.io/
 
 - [Generic Camera](https://www.home-assistant.io/integrations/generic/), setup via GUI
 - [HomeKit Camera](https://www.home-assistant.io/integrations/homekit_controller/)
-- [ONVIF](https://www.home-assistant.io/integrations/onvif/)
+- [ONVIF](https://www.home-assistant.io/integrations/onvif/) via [Module: ONVIF](#module-onvif)
```

Comment:
This is the wrong section, this one is talking about importing cameras FROM Home Assistant, not TO Home Assistant. Please revert this.

Here is where I meant:

https://github.com/AlexxIT/go2rtc#module-hass:~:text=Integrations%20%3E%20Add%20Integration-,%3E%20ONVIF%20%3E,-Host%3A%20127.0.0.1

![image](https://github.com/user-attachments/assets/d287427d-f928-4f20-8814-24567403152f)